go/logger/standard.go

```diff
@@ -350,3 +349,15 @@ func PickFirstError(errors ...error) error {
 func (log *Standard) SetExternalHandler(handler ExternalHandler) {
 	log.externalHandler = handler
 }
+
+type UnforwardedLogger Standard
+
+func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
+	return (*UnforwardedLogger)(log)
+}
+
+func (log *UnforwardedLogger) Debug(s string, args ...interface{})   { log.internal.Debug(s, args...) }
+func (log *UnforwardedLogger) Error(s string, args ...interface{})   { log.internal.Error(s, args...) }
+func (log *UnforwardedLogger) Warning(s string, args ...interface{}) { log.internal.Warning(s, args...) }
+func (log *UnforwardedLogger) Info(s string, args ...interface{})    { log.internal.Info(s, args...) }
+func (log *UnforwardedLogger) Profile(s string, args ...interface{}) { log.internal.Debug(s, args...) }
```

Comment:
Though looking at rotatelog_nix.go, I think we're doing the rotation by `dup2`ing stdout and stderr to be different file handles. In that case, maybe it doesn't matter what logger instance we're using?